exchanges/bitmex/bitmex_types.go

```diff
@@ -291,18 +291,18 @@ type Order struct {
 	ClOrdID               string    `json:"clOrdID"`
 	ClOrdLinkID           string    `json:"clOrdLinkID"`
 	ContingencyType       string    `json:"contingencyType"`
-	CumQty                int64     `json:"cumQty"`
+	CumQty                float64   `json:"cumQty"`
```

Comment:
From bitmex doc, CumQty and LeavesQty has a "Number" type, is not very make sense to assign it as int64 in crypto trading environment.

And TransactTime has same format as Timestamp from the doc: 
"transactTime": "2021-10-02T06:10:25.230Z",
"timestamp": "2021-10-02T06:10:25.230Z"

https://www.bitmex.com/api/explorer/#!/Order/Order_getOrders